libass/ass.h

```diff
@@ -62,9 +62,11 @@ extern "C" {
 typedef struct ass_image {
     int w, h;                   // Bitmap width/height
     int stride;                 // Bitmap stride
-    unsigned char *bitmap;      // 1bpp stride*h alpha buffer
+    unsigned char *bitmap;      // 1-byte/pixel alpha buffer
                                 // Note: the last row may not be padded to
                                 // bitmap stride!
+                                // The guaranteed allocated size is
+                                // `(stride * (h - 1)) + w`
```

Comment:
Perhaps also mention anything outside the actual `w`×`h` area may be uninitialised even if allocated?